### ISSUE TITLE
false positive error on `crate::macro!()` macro call

### ISSUE THREAD

**Issue by WaffleLapkin** [intellij-rust/intellij-rust#5163]
## Environment

* **IntelliJ Rust plugin version:** 0.2.120.3030-193-nightly
* **Rust toolchain version:** rustc 1.42.0 (b8cedc004 2020-03-09)
* **IDE name and version:** Clion, `2019.3.5`
* **Operating system:** Arch linux

## Problem description

Rust plugin shows an error on code that is compilable since rust `1.31.0` (first 2018-edition stable release):

![image](https://user-images.githubusercontent.com/38225716/78056880-7bee8180-738e-11ea-9f06-d655e0fe302a.png)

## Steps to reproduce

Copy-paste this snippet into a newly created project:

```rust
#[macro_export]
macro_rules! macro_name {
	() => {}
}

crate::macro_name!();

fn main() {
    println!("Hello, world!");
}
```

**Comment by dima74** [comment]
@WaffleLapkin looks like it is fixed now:

![image](https://user-images.githubusercontent.com/6505554/103302488-dcd9e180-4a14-11eb-9445-f791b02f4895.png)

* **IntelliJ Rust plugin version:** 0.3.139.3573-203-nightly
* **Rust toolchain version:** 1.48.0 (7eac88abb 2020-11-16) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2020.3 (CL-203.5981.166)
* **Operating system:** Linux 5.9.14-arch1-1
* **Macro expansion engine:** new
* **Name resolution engine:** old
